lib/matplotlib/axes/_base.py

```diff
@@ -2817,24 +2817,57 @@ def get_xlim(self):
         """
         return tuple(self.viewLim.intervalx)
 
-    def _validate_converted_limits(self, limit, convert):
-        """
-        Raise ValueError if converted limits are non-finite.
-
-        Note that this function also accepts None as a limit argument.
-
-        Returns
-        -------
-        The limit value after call to convert(), or None if limit is None.
-
-        """
-        if limit is not None:
-            converted_limit = convert(limit)
-            if (isinstance(converted_limit, float) and
-                    (not np.isreal(converted_limit) or
-                        not np.isfinite(converted_limit))):
-                raise ValueError("Axis limits cannot be NaN or Inf")
-            return converted_limit
+    def _set_lim(self, axis, low, high, low_name, high_name,
+                 target_obj, target_attr, emit, auto, kw):
+        """Helper factoring the functionality of `get_{x,y,z}lim`."""
```

Comment:
This is a private method, but I still think it would be good to write quick one line descriptions of all the args.

Reply:
done